app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
- You can use the same suggestion I made in post: `after_save :update_likes_counter`